assets/css/admin/style.scss

```diff
@@ -2498,6 +2498,15 @@ span.edd-to-and-from--separator {
 	clear: both;
 }
 
+#edd-send-test-summary-save-changes-notice .notice p {
```

Comment:
The CSS is okay but I'm not sure it's actually needed. If you add the `.notice` class to the `#edd-send-test-summary....` element you should be okay with just a paragraph inside, and save a little bit of markup. Probably do `notice inline notice-info` and it will have the blue border instead of the gray one.